src/python/AsyncStageOut/PublisherWorker.py

```diff
@@ -639,26 +639,23 @@ def format_file_3(self, file):
         return nf
 
 
-    def migrateByBlockDBS3(self, workflow, migrateApi, destReadApi, sourceApi, inputDataset, inputBlocks = None):
+    def migrateByBlockDBS3(self, workflow, migrateApi, destReadApi, sourceApi, dataset, blocks = None):
         """
         Submit one migration request for each block that needs to be migrated.
-        If inputBlocks is missing, migrate the full dataset.
-        :return: - the output of destReadApi.listDatasets(dataset = inputDataset)
-                   if all migrations have finished ok,
-                 - an empty list otherwise.
+        If blocks is missing, migrate the full dataset.
```

Comment:
What does it mean "If block is missing, migrate the full dataset" ?
Since you went through the trouble to change variable name to make code more clear, may as well also remove (or clarify) this odd statement